libs/core-ui/src/lib/components/ExpandableText.tsx

```diff
@@ -6,7 +6,7 @@ import { Text, Link, ITextProps, Icon } from "office-ui-fabric-react";
 import React from "react";
 
 export interface IExpandableTextProps extends ITextProps {
-  expandedText?: string;
+  expandedText?: string | React.ReactNode;
```

Comment:
Hmm, that's an odd type... I think this is all related to bold and normal text being mixed. Perhaps something like this is a more elegant solution? https://stackoverflow.com/questions/34778181/styling-text-in-localization-files-bold-italic-etc

Reply:
yes it was for above, I just removed it